src/Files.App/Views/Settings/AdvancedPage.xaml

```diff
@@ -98,6 +98,17 @@
 				</ToggleSwitch>
 			</local:SettingsBlockControl>
 
+			<!--  Leave App Running  -->
+			<local:SettingsBlockControl Title="{helpers:ResourceString Name=SettingsLeaveAppRunning}" HorizontalAlignment="Stretch">
```

Comment:
Should this be under experimental? If you feel it's stable enough, we can keep it here.